.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,24 @@
-[comment]: # (Please replace ... with your information. Remove < and >)
 [comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)
-
-If you submit PR for issue with bounty then write here Fixes #NN where NN is issue number
-
-*otherwise*
-
 fixes #...
 
 ### Summary
-
 [comment]: # (Summarise the problem and how the pull request solves it)
 ...
 
-<!-- (Optional, remove if no changes to documentation) -->
-Documentation change PR (review please): https://github.com/status-im/status.im/pull/xxx
-
 ### Review notes
-<!-- (Optional. Specify if something in particular should be looked at, or ignored, during review) -->
+[comment]: # (Optional. Specify if something in particular should be looked at, or ignored, during review)
 
 ### Testing notes
-<!-- (Optional) -->
+[comment]: # (Optional)
 
 #### Platforms
-<!-- (Optional. Specify which platforms should be tested) -->
+[comment]: # (Optional. Specify which platforms should be tested)
 
 - Android
 - iOS
-- macOS
```

Comment:
desktop platforms aren't actual in `status-mobile` repo

Reply:
I think this is related to changes in dev exp.

But yeah, they aren't common, let's remove it

Reply:
@ulisesmac, I think that was added back in times when it was possible to build desktop version of status-mobile using `react-native-desktop-qt`. That was long ago)